### ISSUE TITLE
release ops-agent module to puppet forge please

### ISSUE THREAD

**Issue by pandrian** [GoogleCloudPlatform/puppet-google-logging#17]

**Comment by arjunasokan-bc** [comment]
agreed, much easier to pull from puppet forge than build another CI/CD pipeline to build from source.

**Comment by pandrian** [comment]
@qingling128 @rambleraptor @jsirianni  for visibility

**Comment by pandrian** [comment]
Is this something that you are planning to do? instead of copying the whole module static in puppet code

**Comment by arjunasokan-bc** [comment]
We actually were planning on using this in Puppet Bolt. So we define the modules there instead of actually downloading them. Probably not a huge deal to include the static module in the codebase but seems silly when many of google's other modules are in forge. I agree if we were using traditional Puppet this would likely be a non issue(except when there are updates which means re-compiling)

**Comment by pandrian** [comment]
Hey @arjunasokan-bc we are using r10k ourselves here, I agree defining their module in puppet will make things a bit more tedious. one issue i noticed is that the module is not called properly, I think puppet 6 has some specific naming convention which this module does not follow. had to change name of module for it to work did this occur to you as well?

**Comment by arjunasokan-bc** [comment]
We haven't started using this yet. Right now we're just triggering a script that installs the ops module. I just saw that this was available and started looking into switching. Not being available in forge might be a blocker, still evaluating.
